src/common/entity/compute_device_name.ts

```diff
@@ -0,0 +1,5 @@
+import type { DeviceRegistryEntry } from "../../data/device_registry";
+
+export const computeDeviceName = (
```

Comment:
We already have a `computeDeviceName` function <https://github.com/home-assistant/frontend/blob/dev/src/data/device_registry.ts#L68>

Reply:
Yeah, in my opinion this function does too much with the entities fallback and translation fallback.

Reply:
entities is optional, we could make the translation fallback optional too. We can als wrap the functions, but we can not have 2 functions with the same name in the code base, that will go wrong with importing and is highly confusing.